schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+id INT,
+name TEXT,
```

Comment:
- [ ] Kindly make sure that name is of varchar data type.

Reply:
- suggested changes made 👍🏽 
![image](https://user-images.githubusercontent.com/94127418/178602608-e5131216-f715-4e9b-9584-e6589f68a882.png)